attributes/windows.rb

```diff
@@ -0,0 +1 @@
+default['firewall']['windows']['default_rules'] = true
```

Comment:
This is the chef attribute used while calling the provider. Defined for this feature so that users may override this in their wrapper cookbooks to their desired value.

Default defined here is to preserve existing behaviour of the cookbook: Add default windows rules besides the ones defined with this provider.